taxonomy/views.py

```diff
@@ -0,0 +1,79 @@
+"""
+Views for the taxonomy app
+"""
+
+from django.shortcuts import render, redirect
+
+from .forms import VocabularyForm
+from learningresources.models import Repository
+from taxonomy.models import Vocabulary
+
+
+def edit(request, vocabulary_id=None):
```

Comment:
`vocabulary_id` should not be optional. I'd remove it now and simplify the function (and urls.py).